AdyenActions/Components/3DS2/3DS2 SDK Adapters/AnyChallengeResult.swift

```diff
@@ -11,7 +11,7 @@ internal protocol AnyChallengeResult {
 
     var sdkTransactionIdentifier: String { get }
 
-    var transactionStatus: String? { get }
+    var transactionStatus: String { get }
```

Comment:
This looks like a breaking change for v4, we may not be able to change this.

Also could be the reason why some jobs are failing.
Did you run the test target fully and did they all pass?

Reply:
It is internal 🤔

Reply:
The type `ADYChallengeResult` is public. Which means that it is a breaking change for checkout.
Discussed with @erenbesel. We can only include a version that doesn't have a breaking change. 

The max version without the breaking change is 2.2.6, which is the existing version.

Reply:
But nothing is braking? Am I missing something?

Reply:
Now that you mentioned it. It is indeed not breaking for the consumer of checkout. 
But only for checkout who is a consumer of the 3ds2sdk. 
@erenbesel WDYT?

Reply:
I have fixed Cocoapods. Will check Carthage..